### PR TITLE
[owners] Add API for syntax check (to be used by Travis)

### PR DESCRIPTION
**Pull request by rcebulko**

### CODE REVIEW

File: owners/bootstrap.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ function bootstrap(logger = console) {
     const ownersBot = new OwnersBot(repo);
 
     const initialized = Promise.all([
-      ownersBot.initTeams(github),
+      // ownersBot.initTeams(github),
```
Comment (danielrozenberg) [inline owners/bootstrap.js]: ?

File: owners/src/repo/virtual_repo.js [hunk 2]
```diff
@@ -102,6 +102,7 @@ module.exports = class VirtualRepository extends Repository {
    */
   async findOwnersFiles() {
     const ownersFiles = await this.github.searchFilename('OWNERS');
+    ownersFiles.splice(3);
```
Comment (danielrozenberg) [inline owners/src/repo/virtual_repo.js]: Magic number?

Comment (rcebulko) [reply owners/src/repo/virtual_repo.js]: Oof™ testing code, didn't mean to commit that

File: owners/bootstrap.js (hunk 1 above)
Comment (rcebulko) [reply owners/bootstrap.js]: That was temporary development code to prevent hitting the API a lot. Fortunately, tests would have caught it :D

**Pull request merged**